controller/workitemtype.go

```diff
@@ -72,15 +72,17 @@ func (c *WorkitemtypeController) Create(ctx *app.CreateWorkitemtypeContext) erro
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
-		witTypeModel, err := appl.WorkItemTypes().Create(
+		witTypeModel, err := appl.WorkItemTypes().CreateWithDates(
 			ctx.Context,
 			*ctx.Payload.Data.Relationships.Space.Data.ID,
 			ctx.Payload.Data.ID,
 			ctx.Payload.Data.Attributes.ExtendedTypeName,
 			ctx.Payload.Data.Attributes.Name,
 			ctx.Payload.Data.Attributes.Description,
 			ctx.Payload.Data.Attributes.Icon,
-			modelFields)
+			modelFields,
+			ctx.Payload.Data.Attributes.CreatedAt,
```

Comment:
> I'm even concerned that the data might be set to null/default value in the DB if the client does not provide the values, to be honest.

Well, that is the normal case actually. If we don't specify a value for `CreatedAt` and `UpdatedAt` those values are `nil` ;)